lib/screens/login_page.dart

```diff
@@ -0,0 +1,132 @@
+import 'dart:convert';
+import 'package:app/screens/home_page.dart';
+import 'package:flutter/material.dart';
+
+import '../API/auth.dart';
+
+// Variables
+bool _passwordVisible = false;
+TextEditingController usernameForm = TextEditingController();
+TextEditingController passwordForm = TextEditingController();
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPage();
+}
+
+@override
+void initState() {
+  _passwordVisible = false;
+}
+
+// Create login Page with username and password
+class _LoginPage extends State<LoginPage> {
+  // Go to next page
+  void toNextPage() {
+    Navigator.pushReplacement(
+      context,
+      MaterialPageRoute(builder: (context) => const HomePage()),
+    );
+  }
```

Comment:
These variables are only used in the `_LoginPage` state. It should be defined there. 
Also,  there is no need to set `_passwordVisible` to false in the `initState()`, that's redundant. 
  

```suggestion

class LoginPage extends StatefulWidget {
  const LoginPage({super.key});

  @override
  State<LoginPage> createState() => _LoginPage();
}

// Create login Page with username and password
class _LoginPage extends State<LoginPage> {
  // Variables
  bool _passwordVisible = false;
  TextEditingController usernameForm = TextEditingController();
  TextEditingController passwordForm = TextEditingController();

  // Go to next page
  void toNextPage() {
    Navigator.pushReplacement(
      context,
      MaterialPageRoute(builder: (context) => const HomePage()),
    );
  }
```